release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4372,3 +4372,34 @@ $ oc adm release info 4.13.50 --pullspecs
 [id="ocp-4-13-50-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.51
+[id="ocp-4-13-51"]
+=== RHSA-2024:6811 - {product-title} 4.13.51 bug fix and security updates
+
+Issued: 25 September 2024
+
+{product-title} release 4.13.51, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6811[RHSA-2024:6811] advisory. TThe RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:6813[RHBA-2024:6813] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.51 --pullspecs
+----
+
+[id="ocp-4-13-51-enhancements"]
+==== Enhancements
+
+* CENTOS 8 has recently ended its lifecycle. This release updates the CENTOS 8 references to CENTOS 9. (link:https://issues.redhat.com/browse/OCPBUGS-39160[*OCPBUGS-39160*])
+
+[id="ocp-4-13-51-bug-fixes"]
+==== Bug fixes
+
+* Previously, `openvswitch` could function by using older configurations after a cluster upgrade operations. These older configurations might have caused issues for networking components that interact with `openvswitch`. With this release, `openvswitch` is restarted after an upgrade operation so that `openvswitch`  is forced to receive the latest configurations. (link:https://issues.redhat.com/browse/OCPBUGS-41723[*OCPBUGS-41723*])
+
+* Previously, the Cluster Resource Override Operator was incorrectly using `kube` version 1.21 clients on {product-title}. The Operator should have been using `kube` version 1.26 clients. The `kube` version 1.21 clients can no longer work because the `flowcontrol.apiserver.k8s.io/v1beta1` API was previously removed. With this release, the Cluster Resource Override Operator now uses `kube` version 1.21 clients so that this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-39236[*OCPBUGS-39236*])
```

Comment:
Does it need to explicitly reference it as "`kube`" like this, or can this just be generically "Kubernetes"? E.g.:
```suggestion
* Previously, the Cluster Resource Override Operator was incorrectly using Kubernetes version 1.21 clients on {product-title}. The Operator should have been using Kubernetes version 1.26 clients. Kubernetes 1.21 clients can no longer work because the `flowcontrol.apiserver.k8s.io/v1beta1` API was previously removed. With this release, the Cluster Resource Override Operator now uses Kubernetes 1.21 clients so that this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-39236[*OCPBUGS-39236*])
```

Reply:
:heavy_check_mark: